dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -441,14 +443,54 @@ const dismissGateAuxia = (
 	setShowGate(false);
 };
 
+const decideIsSupporter = (): boolean => {
+	// nb: We will not be calling the Auxia API if the user is signed in, so we can set isSignedIn to false.
+	const isSignedIn = false;
+	const is_supporter = hasSupporterCookie(isSignedIn);
```

Comment:
I think you want to use `shouldHideSupportMessaging()` rather than `hasSupporterCookie()` right now. They will be the same eventually but currently `hasSupporterCookie` doesn't include one off contributors

Reply:
Updated. Thanks!